src/main/resources/openapi/sqs/Models/RejectedReason.yaml

```diff
@@ -10,4 +10,9 @@
       type: array
       items:
         type: string
-        maxLength: 100
\ No newline at end of file
+        maxLength: 100
+    reasonList:
+      type: array
+      description: The list of reasons the ERO rejected the application
+      items:
+        $ref: './RejectedReasonItem.yaml'
```

Comment:
Given this model is only used to communicate between EROP (postal and proxy APIs) and this service, I don't think we need to include the deprecated `reasons` property, given it's derivable from `reasonList`